task.go

```diff
@@ -0,0 +1,18 @@
+package libcarina
+
+type Input interface{}
```

Comment:
@rgbkrk So, out of curiosity, what issues are you seeing in the future using `interface{}` here?